### DISCUSSION TITLE
Better PDF embedding

### DISCUSSION THREAD

**Discussion by Its4Nik** [outline/outline#7742] [Ideas]
I have to use a lot of PDFs with Outline since i am using it as a notetaking app for school and some github projects.

And for now i have to use a markdown to PDF converter which kinda works but still is a pain in the ***.

Thats why I am proposing a feature for a new command `/pdf` where we can upload a PDF and it will get renderd using the browser integrated PDF rendering capabilities. Maybe this would also make PDF editing a whole lot easier then downloading, editing, converting, inserting...

**Comment by whuwangyong** [comment]
Try logseq or siyuan-note, they do pdf well.

**Comment by tommoor** [comment]
The UX of embedding docs in docs is kind of ugly though as you have a double scroll area, It would be nice if the PDF could open in it's own viewer like an image does though?

**Reply by Its4Nik** [reply]
Yes that would also be great!
